src/pages/Editor/sections/OutputComponent/Output.js

```diff
@@ -5,7 +5,8 @@ import { EditContext } from "../../containers/editContext";
 
 const OutputComponent = () => {
   const editContext = useContext(EditContext);
-  const page = require('./ruled1.png');
+  // const page = require('./ruled1.png');
+  const page = require(`./${editContext.pageSrc}`);
  console.log(`${editContext.pageSrc}`);
 
```

Comment:
Try to add some other pages as well which may be useful for students

Reply:
Can you provide pages you want me to add?

Reply:
😅 I have to do a review of the common requirements. Let me check if I have the last year's report for this, I'll send you the images then